### ISSUE TITLE
How to set the PERMITTED_MEMORY in synthetic_short_gene_example\single_cell_calculations_short_gene.py

### ISSUE THREAD

**Issue by hongpengzhou** [ManchesterBioinference/burstInfer#1]
Hi, I encounter a problem when I set the PERMITTED_MEMORY in synthetic_short_gene_example\single_cell_calculations_short_gene.py. The released code set this value to be 8. But when I set it as 5 or 6, the error appears. Do I have to set the PERMITTED_MEMORY as the powers of base 2, e.g., 1 = 2^0, 2 = 2^1, 4 = 2^2, 8 = 2^3....

**Comment by jbqbb** [comment]
Yes, that's right - PERMITTED_MEMORY should always be powers of base 2. If you set it to any other value then the code will crash. The permitted memory is the maximum size that the truncated state space can take (it's not relevant to the full model). It should be set to the maximum value that is computationally feasible - basically, the longest time to train the model that you can get away with. Each time step the state space expands until it reaches the maximum allowed size (see Figure 3 in the paper).

As an aside: 8 is actually a very small state space that was just used for the toy example with a very short gene. Real data in e.g. Drosophila would need something longer, but it depends on the system you are looking at.

Performance should convergence towards the full model as you add more allowed states.
